hosts/mno001/network.nix

```diff
@@ -37,33 +37,55 @@ in
         bondConfig = {
           Mode = "802.3ad"; # LACP 
           MIIMonitorSec = "250ms";
-          LACPTransmitRate = "fast";
+          LACPTransmitRate = "fast"; 
         };
       };
 
-      "20-svc-management" = {
+      # 20-svc-internet
+      "20-uplink" = {
         netdevConfig = {
-          Name = "svc-management";
+          # svc-internet
+          Name = "uplink";
           Kind = "vlan";
         };
         vlanConfig = {
-          Id = 102;
+          Id = 100;
         };
       };
 
-      "20-uplink" = {
+      "20-br-svc-internet".netdevConfig = {
+        Name = "br-svc-internet";
+        Kind = "bridge";
+      };
+
+      "20-svc-services" = {
         netdevConfig = {
-          Name = "uplink";
+          Name = "svc-services";
           Kind = "vlan";
         };
         vlanConfig = {
-          Id = 100;
+          Id = 101;
+        };
+      };
+
+      "20-br-svc-services".netdevConfig = {
+        Name = "br-svc-services";
+        Kind = "bridge";
+      };
+
+      "20-svc-management" = {
+        netdevConfig = {
+          Name = "svc-management";
+          Kind = "vlan";
+        };
+        vlanConfig = {
+          Id = 102;
         };
       };
 
-      "30-microvm-inet".netdevConfig = {
+      "20-br-svc-management".netdevConfig = {
+        Name = "br-svc-management";
```

Comment:
If the vlan subinterface has a generic naming we can keep the VLAN name for the bridges (read: `svc-management`).